src/main/java/com/rakovets/course/java/core/example/exception_handling/task3/Task3.java

```diff
@@ -0,0 +1,11 @@
+package com.rakovets.course.java.core.example.exception_handling.task3;
+
+public class Task3{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"